src/NonCumulative/Soundness/LogRel.agda

```diff
@@ -263,14 +263,15 @@ record _⊢_∶_®↓[_]_∈El_ Γ t T i c (A≈B : A ≈ B ∈ 𝕌 i) : Set wh
     c∈⊥  : c ∈′ Bot
     krip : Δ ⊢w σ ∶ Γ → let u , _ = c∈⊥ (len Δ) in Δ ⊢ t [ σ ] ≈ Ne⇒Exp u ∶[ i ] T [ σ ]
 
-record _⊢_∶_®↑[_]_∈El_ Γ t T i a (A≈B : A ≈ B ∈ 𝕌 i) : Set where
+record _⊢_∶_®↑[_]_∈El_≈_ Γ t T i a A B : Set where
   field
     t∶T  : Γ ⊢ t ∶[ i ] T
+    A≈B  : A ≈ B ∈ 𝕌 i 
```

Comment:
I thought it didn't work well. why bring it back?

Reply:
it works for the `®El⇒®↑El`. For `®↓El⇒®El`, I keep `_⊢_∶_®↓[_]_∈El_` unchanged